e2e/Makefile

```diff
@@ -0,0 +1,24 @@
+IMG=secret-manager-e2e:test
+
+start-kind:
+	kind create cluster \
+	  --name secret-manager \
+	  --config kind.yaml \
+	  --retain \
+	  --image "kindest/node:v1.19.0"
+
+test: e2e-image
+	$(MAKE) -C ../ docker-build IMG=secret-manager:test
+	kind load docker-image --name="secret-manager" secret-manager:test
+	kind load docker-image --name="secret-manager" secret-manager-e2e:test
+	./run.sh
+
+e2e-bin:
+	ginkgo build .
```

Comment:
This required `CGO_ENABLED=0` on my system (wsl2), should be added unless there is a reason not to.